modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java

```diff
@@ -73,4 +80,23 @@ public SELF withPassword(final String password) {
         this.password = password;
         return self();
     }
+
+    @Override
+    protected void waitUntilContainerStarted() {
+        String regEx = ".*database system is ready to accept connections.*\\s";
+        logger().info("Waiting for database to log '{}'", regEx);
+
+        WaitingConsumer waitingConsumer = new WaitingConsumer();
+        this.followOutput(waitingConsumer);
+
+
+        Predicate<OutputFrame> waitPredicate = outputFrame ->
+                outputFrame.getUtf8String().matches(regEx);
```

Comment:
This is an out of the way way to mention this, but I've had this kind of code throw a NullPointerException when the regEx doesn't match any output.  Pretty sure it's because getUtf8String [returns null](https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/containers/output/OutputFrame.java#L33).  I don't see the harm in having that return an empty string instead of null, but haven't tested it.  With the code the way it is, I think we could use a null check here.